iOS/G3MiOSSDK/Commons/Mesh/AbstractMesh.hpp

```diff
@@ -46,7 +50,10 @@ class AbstractMesh : public Mesh {
                const IFloatBuffer* colors,
                const float colorsIntensity,
                bool depthTest,
-               const IFloatBuffer* normals);
+               const IFloatBuffer* normals,
+               bool polygonOffsetFill = false,
+               float polygonOffsetFactor = 0,
+               float polygonOffsetUnits = 0);
 
```

Comment:
Please don't used default values in this constructor.  They should be on the constructors of the concrete subclasses.